tuf/api/serialization/util.py

```diff
@@ -0,0 +1,149 @@
+"""Utility functions to facilitate TUF metadata de/serialization.
+
+Currently, this module contains functions to convert between the TUF metadata
+class model and a corresponding dictionary representation.
+
+"""
+
+
+from typing import Any, Dict, Mapping
+
+from tuf import formats
+from tuf.api.metadata import (Signed, Metadata, Root, Timestamp, Snapshot,
+                              Targets)
+
+
+def _get_signed_common_args_from_dict(_dict: Mapping[str, Any]) -> list:
```

Comment:
How did you decide to accept `typing.Mapping` instead of `typing.Dict` here and elsewhere in this file?

Reply:
Good question! The [`typing.Dict` docs](https://docs.python.org/3/library/typing.html#typing.Dict) have the answer:
> To annotate arguments it is preferred to use an abstract collection type such as Mapping.

I didn't really question that advice, but it makes sense when you consider the API design principle of returning specific types, while accepting generic types.